proposal/numbers.md

```diff
@@ -0,0 +1,104 @@
+# Floating Point Numbers: Draft 1.0
+
+*[(Issue)](https://github.com/sass/sass/issues/2892)*
+
+This proposal formally defines all numbers in Sass to be double-precision or 64-bit floating point numbers.
+
+## Table of Contents
+
+- [Floating Point Numbers: Draft 1.0](#floating-point-numbers-draft-10)
+  - [Table of Contents](#table-of-contents)
+  - [Background](#background)
+  - [Division by Zero](#division-by-zero)
+  - [Equality](#equality)
+  - [Ordering](#ordering)
+  - [Hashing](#hashing)
+  - [Formatting](#formatting)
+
+## Background
+
+Numbers in Sass today are ill-defined. Between implementations there
+is very little agreement around edge cases, and there are few spec tests
+exercising these cases.
+
+The behavior of `dart-sass` and `libsass`, the two officially "blessed" implementations
+at the time of writing, differ greatly with regard to numeric equality, rounding, and overflows.
+
+Additionally, this lack of definition has caused both [regressions](https://github.com/sass/dart-sass/issues/807)
+and [crashes](https://github.com/sass/dart-sass/issues/1059) for `dart-sass`.
+
+This document serves to unify these edge cases and resolve existing issues within implementations.
+
+## Division by Zero
+
+Any positive number divided by zero will return `Infinity`
+
+Zero divided by zero will return `NaN`
+
+Any negative number divided by zero will return `-Infinity`
+
+## Equality
+
+Numbers in Sass are symmetrically and transitively equal. That is,
+if `a == b`, then `b == a` and if `a == b` and `b == c`, then `a == c`.
+
+Numbers in Sass are *not* necessarily reflexively equal. `NaN == NaN`,
+`Infinity == Infinity`, and `-Infinity == -Infinity` will always be false.
+
+In the case that two real numbers use more than 10 decimal places of precision,
+both numbers are truncated to 10 decimal places and compared.
```

Comment:
It would probably be good to include non-normative examples here. (We use Markdown block quotes to indicate a non-normative that appears within a normative section.)